src/main/java/core/basesyntax/Ball.java

```diff
@@ -0,0 +1,29 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Ball {
+    private Color color = ColorSupplier.getRandomColor();
+    private int number = new Random().nextInt(100);
+
+    public int getNumber() {
+        return number;
+    }
+
+    public Color getColor() {
+        return color;
+    }
+
+    public void setNumber(int number) {
+        this.number = number;
+    }
+
+    public void setColor(Color color) {
+        this.color = color;
+    }
+
+    @Override
+    public String toString() {
+        return "Ball color: " + this.color + ". Ball number: " + this.number;
```

Comment:
when we write `someObject + someString`, then `someObject.toString()` will be invoked automatically
[Use name() for getting String representation of enum constants](https://mate-academy.github.io/jv-program-common-mistakes/java-core/oop/lottery#use-name-for-getting-string-representation-of-enum-constants)